### PR TITLE
UserGuide.adoc: Add Import command

### LINKED ISSUE #62: Update User guide documentation for Import feature (Week 7)
Update User guide documentation for Import feature by Week 7

### PR DESCRIPTION
**Pull request by Exeexe93**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -157,6 +157,35 @@ There is no need to save manually.
...
+=== Listing orders : `list`
+
+[None]
+==== Description
+[none]
+* You can request to list the delivery order to see.
+The done status of the delivery order will be used to list the delivery orders which have the same status.
+There are three types of list commands that are shown in the examples below.
+
+
```
Comment (khsc96) [inline docs/UserGuide.adoc:188]: Might be this additional lines failing the checkstyle test

**Review by khsc96 — CHANGES_REQUESTED** [review]
Please do find out where is the checkstyle problem and amend it. Other than checkstyle LGTM

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc:188]: Not that problem. The problem lies with there are no newline at the end of the file. Problem solved by adding newline at the end.

**Review by khsc96 — APPROVED** [review]
LGTM

**Review by Cherweijie — APPROVED** [review]
LGTM.

**Pull request merged**